src/net/udp.rs

```diff
@@ -576,9 +578,11 @@ impl UdpSocket {
     ///
     /// The number of bytes written is returned, if successful, or an error is
     /// returned otherwise. If the socket is not currently writable then a
-    /// "would block" error is returned. This operation does not block.
+    /// [`WouldBlock`][link] error is returned. This operation does not block.
```

Comment:
For future reference you can use:

```
... text .. [`WouldBlock`] ... more text ...

[`WouldBlock`]: https://doc.rust-lang.org/nightly/std/io/enum.ErrorKind.html#variant.WouldBlock
```

But I'll merge this as is.

Reply:
Yes, that looks better. Thank you for your instruction!